src/sql/workbench/parts/commandLine/electron-browser/commandLine.ts

```diff
@@ -130,6 +153,48 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution
 		}
 	}
 
+	async handleURL(uri: URI): Promise<boolean> {
```

Comment:
Unit tests? Seems like it should be simple to do some basic tests with urls we know we will get from callers.